Makefile

```diff
@@ -511,6 +511,30 @@ tests/foundry/out/kompiled/foundry.k.prove: tests/foundry/out/kompiled/timestamp
 	    $(KEVM_OPTS) $(KPROVE_OPTS)                                      \
 	    $(addprefix --exclude-test , $(shell cat tests/foundry/exclude))
 
+foundry_golden := tests/foundry/golden
+foundry_diff_tests := $(shell cat tests/foundry/checkoutput)
+
+test-foundry-kcfg-diff: $(patsubst %, $(foundry_golden)/%.check, $(foundry_diff_tests))
+
+foundry-fail: tests/foundry/out/kompiled/timestamp
```

Comment:
Need ot add tese targets to `.PHONY` at the top of the file. That's because they aren't actually file-based targets, and `make` treats non-file and file-based targets differently.